docs/tutorial/part-three/index.md

```diff
@@ -1,57 +1,57 @@
+लेआउट कौम्पोनॅन्टस आपकी साइट के उन हिस्सों के लिए हैं जिन्हें आप कई पेजेज़ में शेयर करना चाहते हैं। उदाहरण के लिए, साइटों में आमतौर पर एक शेयर्ड हेडर और फुटर के साथ एक लेआउट कौम्पोनॅन्टस होगा। लेआउट में जोड़ने के लिए अन्य सामान्य चीजें एक साइडबार और / या नेविगेशन मेनू हैं। उदाहरण के लिए इस पेज पर, सबसे ऊपर का हेडर gatsbyjs.org के लेआउट कौम्पोनॅन्ट का हिस्सा है।
 
-Let's dive into part three.
+चलिए भाग तीन शुरू करते हैं।
 
-## Using plugins
+## प्लगइन्स का उपयोग करना
 
-You’re probably familiar with the idea of plugins. Many software systems support adding custom plugins to add new functionality or even modify the core workings of the software. Gatsby plugins work the same way.
+आप शायद प्लगइन्स के विचार से परिचित हैं। कई सॉफ्टवेयर सिस्टम नई कार्यक्षमता जोड़ने या सॉफ़्टवेयर के मुख्य कामकाज को संशोधित करने के लिए कस्टम प्लगइन्स को जोड़ने का समर्थन करते हैं। Gatsby प्लगइन्स उसी तरह काम करते हैं।
 
-Community members (like you!) can contribute plugins (small amounts of JavaScript code) that others can then use when building Gatsby sites.
+समुदाय के सदस्य (आप की तरह!) प्लगइन्स (जावास्क्रिप्ट कोड की छोटी मात्रा) का योगदान कर सकते हैं जो अन्य लोग तब उपयोग कर सकते हैं जब Gatsby साइटों का निर्माण करते हैं।
 
-> There are already hundreds of plugins! Explore the Gatsby [Plugin Library](/plugins/).
+> पहले से ही सैकड़ों प्लगइन्स हैं! गैट्सबी [प्लगइन लाइब्रेरी] (/pluginw/) का अन्वेषण करें।
 
-Our goal with plugins is to make them straightforward to install and use. You will likely be using plugins in almost every Gatsby site you build. While working through the rest of the tutorial you’ll have many opportunities to practice installing and using plugins.
+प्लगइन्स के साथ हमारा लक्ष्य उन्हें स्थापित करने और उपयोग करने के लिए सीधा करना है। आप संभवतः आपके द्वारा बनाए गए लगभग हर गैट्सबी साइट में प्लगइन्स का उपयोग कर रहे होंगे। बाकी ट्यूटोरियल के माध्यम से काम करते समय आपके पास प्लगइन्स को स्थापित करने और उपयोग करने के कई अवसर हैं।
 
-For an initial introduction to using plugins, we'll install and implement the Gatsby plugin for Typography.js.
+प्लगइन्स का उपयोग करने के लिए प्रारंभिक परिचय के लिए, हम टाइपोग्राफी के लिए Gatsby प्लगइन को स्थापित और कार्यान्वित करेंगे।
 
-[Typography.js](https://kyleamathews.github.io/typography.js/) is a JavaScript library which generates global base styles for your site's typography. The library has a [corresponding Gatsby plugin](/packages/gatsby-plugin-typography/) to streamline using it in a Gatsby site.
+[टाइपोग्राफी.जेएस] (https://kyleamathews.github.io/typography.js/) एक जावास्क्रिप्ट लाइब्रेरी है जो आपकी साइट की टाइपोग्राफी के लिए वैश्विक आधार शैली उत्पन्न करती है। लाइब्रेरी में एक Gatsby साइट का उपयोग करके इसे स्ट्रीमलाइन करने के लिए [संबंधित गैट्सबी प्लगइन] (/ संकुल / gatsby- प्लगइन-टाइपोग्राफी /) है।
```

Comment:
```
[टाइपोग्राफी.जेएस] (https://kyleamathews.github.io/typography.js/)
```
=>
```
[Typography.js] (https://kyleamathews.github.io/typography.js/)
```

Reply:
```
[Typography.js] (https://kyleamathews.github.io/typography.js/)
```

remove extra space before brackets.

Sorry, it was missed from my end, I forgot to remove space before